ui/tests/integration/components/pki/config-pki-test.js

```diff
@@ -11,84 +11,91 @@ const component = create(configPki);
 module('Integration | Component | config pki', function (hooks) {
   setupRenderingTest(hooks);
 
-  hooks.beforeEach(function () {
+  hooks.beforeEach(async function () {
     this.owner.lookup('service:flash-messages').registerTypes(['success']);
-  });
-
-  const config = function (saveFn) {
-    return {
-      save: saveFn,
-      rollbackAttributes: () => {},
-      tidyAttrs: [
-        {
-          type: 'boolean',
-          name: 'tidyCertStore',
-        },
-        {
-          type: 'string',
-          name: 'anotherAttr',
-        },
-      ],
-      crlAttrs: [
-        {
-          type: 'string',
-          name: 'crl',
-        },
-      ],
-      urlsAttrs: [
-        {
-          type: 'string',
-          name: 'urls',
-        },
-      ],
+    this.store = this.owner.lookup('service:store');
+    this.config = await this.store.createRecord('pki/pki-config');
```

Comment:
instead of manually setting the attrs as we were previously - I wanted to use the real model to make sure defaults are correct and the model updates correctly